tests/features/bootstrap/FeatureContext.php

```diff
@@ -39,6 +39,7 @@ class FeatureContext extends RawDrupalContext implements SnippetAcceptingContext
    * Define ASCII values for key presses.
    */
   const KEY_LEFT = 37;
+
```

Comment:
If it is important to have a blank line here then a new docblock should be added for the second line, and the docblock for the first line should be rewritten.